pkgs/development/python-modules/pyvcf/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, pytest
+}:
+
+buildPythonPackage rec {
+
+  pname = "PyVCF";
+  version = "0.6.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    extension = "tar.gz";
```

Comment:
This line is not needed, "tar.gz" is used by default.

Reply:
Thanks, I didn't know that. Removed it.